src/routers/RouteRouter.js

```diff
@@ -63,20 +64,62 @@ class RouteRouter extends AppDevRouter<Array<Object>> {
         const errors = [];
 
         try {
-            busRoute = await axios.get(`http://${process.env.GHOPPER_BUS || 'ERROR'}:8988/route`, {
-                params: parameters,
-                paramsSerializer: (params: any) => qs.stringify(params, { arrayFormat: 'repeat' }),
-            });
+            const options = {
+                method: 'GET',
+                url: `http://${process.env.GHOPPER_BUS || 'ERROR'}:8988/route`,
+                qs:
+                    {
+                        vehicle: 'pt',
+                        weighting: 'short_fastest',
+                        elevation: false,
+                        point: [start, end],
+                        points_encoded: false,
+                    },
+            };
+
+            busRoute = JSON.parse(await new Promise((resolve, reject) => {
+                request(options, (error, response, body) => {
```

Comment:
@zeladada thoughts on wrapping all of this `new Promise(...)` into its own function so you can just do `await createRequest(url, options)`?

Would get rid of duplicated code

Reply:
I will work on that. Where should this function be created? Would you recommend creating a separate `utils/requestUtils.js` file that has all the common request functionalities

Reply:
yep that makes sense to me